react/src/components/FolderExplorer.tsx

```diff
@@ -78,7 +85,7 @@ const FolderExplorer: React.FC<Props> = ({
           // if (res.status === 404) {
           //   return null;
           // }
-          return res.items;
+          return res.items as Array<FileItem>;
```

Comment:
If there is no type inference for response like `TanQuery` or `suspenseTanQuery`, is it okay to understand that you create a separate interface and specify the return data type via as, and if the response type of that API changes in the future (which rarely happens now because it goes to relay), change the interface to maintain the type too?

Reply:
I changed a way to define result type. Please check it. 
Yes. Currently, changes in the API response types have to be managed manually.